### ISSUE TITLE
add info for remote sessions!

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-presentations2all#13]
e.g., some of the info here https://www.w3.org/WAI/APA/task-forces/research-questions/wiki/Accessibility_of_Remote_Meetings#Meeting_Hosting_and_Participation

Update: That morphed into this early draft [Accessibility of Remote Meetings](https://w3c.github.io/apa/remote-meetings/)

**Comment by iadawn** [comment]
Certainly topical and likely to be more relevant going forward as people have realised that remote sessions are a viable thing!

**Comment by nitedog** [comment]
We now have a lot of tips in the AV Tutorial as well, so it may be more about cross-referencing? Not sure what belongs where, though.

**Comment by eoncins** [comment]
Agree but the information provided in https://www.w3.org/WAI/APA/task-forces/research-questions/wiki/Accessibility_of_Remote_Meetings#Meeting_Hosting_and_Participation is incomplete.
No mention is made to: 
- Provide participants with alternative ways to access information in advance.
- Keyboard navigation, screen reader compatibility, audio description (etc...) 
Also recommendations such as "Prepare documents, presentations, multimedia and other materials so as to conform to Web Content Accessibility Guidelines (WCAG) 2.1, preferably at Level AA or beyond" are too vague and do not provide guidance to users/participants.

**Comment by shawna-slh** [comment]
See also [Accessible Virtual Conferences](https://www.sigaccess.org/accessible-virtual-conferences/)

(Thanks, Michele! https://github.com/w3c/wai-presentations2all/issues/60 )

**Comment by shawna-slh** [comment]
My previous thoughts on how info from [old rough notes](https://www.w3.org/WAI/APA/task-forces/research-questions/wiki/Accessibility_of_Remote_Meetings#Meeting_Hosting_and_Participation) might be integrated into [Make Presentations Accessible](https://www.w3.org/WAI/teach-advocate/accessible-presentations/) "existing doc":

•	Prepare documents, presentations, multimedia and other materials so as to conform to Web Content Accessibility Guidelines (WCAG) 2.1, preferably at Level AA or beyond.
**_in existing doc:_ complete? partial**
•	Ensure that the files containing such documents, presentations, multimedia, etc., are available directly to meeting participants, and that screen sharing is not the only means of obtaining these materials.
**_in existing doc:_ maybe not**
•	Be prepared to engage a captioning service, or to provide captions.
**_in existing doc:_ yes – could be more direct with “captions” in main point**
•	Be prepared to engage a sign language interpretation service, or to provide sign language interpretation.
**_in existing doc:_ yes – could be more direct with “sign language” in main point**
•	Be prepared to provide alternatives to any aspects of the remote meeting platform that are not accessible to meeting participants. For example, if a tool used to coordinate turn taking in a meeting (e.g., a "hand raising" control) is not accessible to keyboard-only users or to users of assistive technologies, offer alternative means of managing turn taking.
**_in existing doc:_ no**
•	Create accessible meeting notes that can be made available to participants after the meeting.
**_in existing doc:_ humm – not really a requirement.**
•	Become familiar with the accessibility features of the meeting software platform. See #References below for a partial list of links to accessibility information offered by well known meeting platform providers. 
**_in existing doc:_ too generic**
•	Address any accessibility issues in the meeting platform. For example, in some platforms, screen reader users cannot override chat interruptions, and when there is a lot of chat, they cannot hear the speakers. 
**_in existing doc:_ add**
•	Request that all participants test their audio and video in advance of any meeting. 
**_in existing doc:_ maybe not axe-specific enough?**
•	Remind all participants to, if possible, ensure that their faces, including their mouths, are visible and well-lit in the videoconferencing window. 
**_in existing doc:_ yes for speakers, not all participants**
•	Request that any participants using virtual backgrounds (photo or video) test the quality of their projection to ensure that there is no flicker of the participants' image. 
**_in existing doc:_ no (and need stronger warning about this)**
•	If the sound degrades during a videoconference, request that participants turn off their video to see if that improves the audio quality. 
**_in existing doc:_ maybe too generic?**
•	If streaming captions or streaming sign language interpreting will be used during a videoconference, make sure to select a videoconferencing platform in which participants can anchor and freely re-size any window in which these communication accommodations will be displayed, independently from the windows showing any content (for instance, slides, whiteboard, etc), or any other speaker. 
**_in existing doc:_ add under platform**
•	Allow participants to use a variety of meeting connection methods (e.g. computer, app, telephone) to maximise accessibility and choice for participants with disabilities. This may also lead to additional in-meeting accessibility considerations (e.g. the need to live caption telephone participants) 
**_in existing doc:_ add**

**Comment by shawna-slh** [comment]
closing this issue for new one: https://github.com/w3c/wai-presentations2all/issues/80
